libs/ledger-live-common/src/families/evm/walletApiAdapter.test.ts

```diff
@@ -1,13 +1,12 @@
+import { Transaction } from "@ledgerhq/coin-evm/types/index";
 import { EthereumTransaction as WalletAPITransaction } from "@ledgerhq/wallet-api-core";
-import { Account } from "@ledgerhq/types-live";
 import BigNumber from "bignumber.js";
 import eth from "./walletApiAdapter";
-import { Transaction } from "./types";
+
+// FIXME: add tests for tx of type 2
 
 describe("getPlatformTransactionSignFlowInfos", () => {
   describe("should properly get infos for ETH platform tx", () => {
-    const dummyAccount = { currency: { family: "ethereum" } } as Account;
-
     test("without fees provided", () => {
       const ethPlatformTx: WalletAPITransaction = {
         family: "ethereum",
```

Comment:
Not here, because we are creating a `WalletAPITransaction`
, that has no concept of `evm` family.
That's where the `walletApiAdapter` comes into play. It's role it to adapt / convert a `wallet-api` model to a `ledger-live one`, and vice versa

cf. the list of families handled by the wallet-api here: https://github.com/LedgerHQ/wallet-api/blob/main/packages/core/src/families/common.ts#L8-L29